sdk/servicebus/azure-servicebus/tests/test_dictmixin_amqp.py

```diff
@@ -0,0 +1,76 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+
+from azure.servicebus.amqp._amqp_message import DictMixin
+
+
+class TestDictMixin:
+
+    def test_contains_haskey(self):
+        model = DictMixin()
+        key = "testkey"
+        value = "testval"
+        model.__setitem__(key, value)
+        #assert model.__contains__(key)
```

Comment:
missed this, sorry! same comment as @jalauzon-msft - should this be uncommented here and in the other sb test file?

Reply:
I copied the code verbatim from the eventhub test to keep the code in sync: (which included commented out line)
https://github.com/Azure/azure-sdk-for-python/blob/0e6b75a44ef4130aae3de32bfff759d332d98f27/sdk/eventhub/azure-eventhub/tests/unittest/test_dict_mixin.py#L17

I could either uncomment or remove it from all tests, however. Up to you if you think its enough to check haskey or if it should also check contains.

Reply:
ah, thanks for the context! It seems I originally commented this out since the `key in model` check right below calls `__contains__` as well, so it doesn't need to be called explicitly. Sorry about the confusion!

Would you be able to remove line 18 from all test files you've added here and change line 19 to the following?
`assert key in model    # calls __contains__`

This should help us avoid confusion in the future. 🙂